templates/listing-lister-sale-confirmation.mjml

```diff
@@ -101,36 +101,13 @@
 
         <mj-list>
           <mj-li font-size="16px" color="#000000" font-family="Red Hat Display">
-            You do not ship the item within the allotted time and do not receive approval for an extension.
-          </mj-li>
-          <mj-li font-size="16px" color="#000000" font-family="Red Hat Display">
-            The delivered item does not match the listed item description. Buyers are asked to video record the unboxing and we routinely route items through trusted parties for quality assurance.
-          </mj-li>
-          <mj-li font-size="16px" color="#000000" font-family="Red Hat Display">
-            The item arrives damaged and you do not provide images that show you followed our <a href="https://satscrap.com/seller-guidelines">packaging guidelines</a>.
-            <em>Be aware that if an item is sent without tracking and signature requirement and the buyer claims to not receive it, we can not distribute proceeds to you.</em>
-          </mj-li>
-        </mj-list>
-
-        <mj-text font-size="16px" color="#000000" font-family="Red Hat Display">
-          To show you followed the packaging guidelines, reply to this email with images of each stage of the packaging process. For example:
-        </mj-text>
-
-        <mj-list>
-          <mj-li font-size="16px" color="#000000" font-family="Red Hat Display">
-            The item in a protective inner box or bubble wrap
-          </mj-li>
-          <mj-li font-size="16px" color="#000000" font-family="Red Hat Display">
-            The protected item secured in the outer box
-          </mj-li>
-          <mj-li font-size="16px" color="#000000" font-family="Red Hat Display">
-            The outer box securely taped
```

Comment:
Only the text was requested to be removed. But it said nothing about the section under (didnt even have that in the copy at all). So i was not sure if it was supposed to be removed, but I can re-add if needed.